shell/egl_window.cc

```diff
@@ -84,18 +79,8 @@ EglWindow::EglWindow(size_t index,
 
   m_egl_surface[index] = create_egl_surface(this, m_egl_window[index], nullptr);
 
-  toggle_fullscreen();
```

Comment:
Yeah, I've already made a comment about this, but reviewing happens patch by patch. I would introduce these changes in first patch probably.